### PR TITLE
Add option to change grafana directory

### LINKED ISSUE #18: Add possibility to adjust the monitoring dashboards default directory

### PR DESCRIPTION
**Pull request by davidsuba**
closes #18 

Signed-off-by: David Suba <david.suba@ifne.eu>

### CODE REVIEW

File: chart/templates/dashboards/dashboards.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ metadata:
 {{- if .Values.grafanaDashboards.labelGrafana }}
     grafana_label: {{ include "grafanaLabelAssignment" . | quote }}
 {{- end }}
+{{- if .Values.grafanaDashboards.destinationOverride }}
+{{ toYaml .Values.grafanaDashboards.destinationOverride | indent 4}}
```
Comment (chess-knight) [inline chart/templates/dashboards/dashboards.yaml:22]: add a space before the curly brackets

File: chart/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: dnation-kubernetes-monitoring
-version: 1.0.20
+version: 1.0.21
 appVersion: 1.0.20
```
Comment (matofeder) [inline chart/Chart.yaml:4]: Increase also `appVersion`. 
See the [docs](https://helm.sh/docs/topics/charts/#helm) :
- appVersion: The version of the app that this contains (optional). This needn't be SemVer.

**Review by matofeder — CHANGES_REQUESTED** [review]

File: chart/templates/dashboards/dashboards.yaml (hunk 1 above)
Comment (davidsuba) [reply chart/templates/dashboards/dashboards.yaml]: done

File: chart/Chart.yaml (hunk 2 above)
Comment (davidsuba) [reply chart/Chart.yaml]: done

**Review by matofeder — APPROVED** [review]

**Review by chess-knight — APPROVED** [review]

**Pull request merged**